src/index.html

```diff
@@ -5,8 +5,438 @@
+            <a class="nav__link" href="tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content content--margin_auto">
+      <div class="catalog">
+        <article class="card catalog__card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Product Image"
+          >
+
+          <h2 class="card__name card__name--text_medium">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code card__code--text_small">
+            Product code: 195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <span class="card__review card__review--text_small">
+              Reviews: 5
+            </span>
+          </div>
+
+          <div class="card__price">
+            <span class="card__text">Price:</span>
+
+            <span class="card__amount card__amount--text_large">
+              $2,199
+            </span>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </article>
+
+        <article class="card catalog__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Product Image"
+          >
+
+          <h2 class="card__name card__name--text_medium">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code card__code--text_small">
+            Product code: 195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <span class="card__review card__review--text_small">
+              Reviews: 5
+            </span>
+          </div>
+
+          <div class="card__price">
+            <span class="card__text">Price:</span>
+
+            <span class="card__amount card__amount--text_large">
+              $2,199
+            </span>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
```

Comment:
You only needed to add data-qa attribute to the 1st card

Reply:
Removed.